examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -71,8 +71,8 @@ void RvcDevice::HandleRvcRunChangeToMode(uint8_t newMode, ModeBase::Commands::Ch
     }
 
     // If we fall through at any point, it's because the change is not supported in the current state.
-    response.status = to_underlying(ModeBase::StatusCode::kGenericFailure);
-    response.statusText.SetValue(chip::CharSpan::fromCharString("This change is not allowed at this time. "));
+    response.status = to_underlying(ModeBase::StatusCode::kInvalidInMode);
+    response.statusText.SetValue(chip::CharSpan::fromCharString("This change is not allowed at this time."));
```

Comment:
NIT: Some strings end with period and some do not

Reply:
Thanks, fixed it.